README.md

```diff
@@ -1,7 +1,7 @@
 # **Starboard Discord Bot :0**
 
 ## This repository is for bugs, issues or suggestions about the bot.
-### Alternatively, you can use the star bug command.
+### Alternatively, you can use the star suggest command.
```

Comment:
Maybe add code formatting to "star suggest"?
```suggestion
### Alternatively, you can use the `star suggest` command.
```

Reply:
no, that's bad

Reply:
ok whatever